Task 3/binary_search/question.md

```diff
@@ -0,0 +1,2 @@
+
+Given a sorted array arr[] of n elements, write a function to search a given element x in arr[].
```

Comment:
You don't need a question.md file under Task 3. You can insert this part in a multiline comment on top of your cpp code.